t/lib/keycloak.lua

```diff
@@ -18,6 +18,57 @@ local http = require "resty.http"
 
 local _M = {}
 
+local function split(text, chunk_size)
+    local s = {}
+    for i=1, #text, chunk_size do
+        s[#s+1] = text:sub(i, i + chunk_size - 1)
+    end
+    return s
+end
+
+local function read_cert(str)
```

Comment:
Already exists in t/read_cert.lua. In addition, I don't understand what this function does, the beginning and end of the certificate file usually exist.